### ISSUE TITLE
Create share by e-mail after invitation accepted

### ISSUE THREAD

**Issue by mirekys** [cs3org/reva#1484]
## Scenario

1. Let's assume we have user **A** who starts the invitation workflow by sending an invite link to user **B**'s e-mail b@example.org he knows.
2. User **B** accepts the invite link by going through mesh directory and logging into his provider of choice
3. User **A** now wants to share something to **B**
4. At this moment, user **A** still knows only the user **B**'s e-mail (b@example.org)

## The problem

User **A** has currently no means of determining `grantee.opaqueid` used in the [invitation sharing workflow](https://codimd.web.cern.ch/s/Sy1ndLaxv#Creating-shares), he doesn't even know which `provider.idp` was used by user **B** to accept the invite.

1. User **A** should be able to create a share for the user **B** by using the e-mail adress used in the invitation workflow:
```
$ cmd/reva/reva -insecure ocm-share-create -grantee b@example.org /home/test
```

_or/and_

2. User **A** should be able to list remote users that accepted the invites he sent. Such accepted invite records should contain some `label`, that helps user **A** to recognize, who actually accepted the invite using which invite link (the label would be the used e-mail address b@example.com in this case). User **A** could then find all necessary sharing details about user **B** (provider & opaqueid) in this listing based on the label he already knows and any additional user info exchanged at the moment the invite was accepted.

**Comment by labkode** [comment]
@mirekys can you start implementing option 2)? This is the foundation

**Comment by mirekys** [comment]
Implemented in #1527
